src/python/pants/backend/build_files/fix/deprecations/subsystem.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.option.option_types import SkipOption
+from pants.option.subsystem import Subsystem
+from pants.util.strutil import softwrap
+
+
+class BUILDDeprecationsFixer(Subsystem):
+    options_scope = "build-deprecations-fixer"
+    name = "BUILD Deprecations Fixer"
```

Comment:
Thoughts on whether `pants.toml` fixes should be included here? Maybe we rename it `pants-deprecatios-fixers`? It seems noisy to have two subsystems: the user doesn't care whether something is in a BUILD file vs TOML file

The backend name would be a little weird being in `build_files`, but I think that's probably not a big deal.

Reply:
I've been thinking about a "meta" backend anyways. Maybe it belongs there?
Other "meta" plugins would be:
- Plugin to find and error on redundant `dependencies` declarations
- (this one)
- ... I swear I had more but can't think of any, lol

Reply:
Just calling the backend `pants` would be rather meta, no? :P

Reply:
> Plugin to find and error on redundant dependencies declarations

Oh, yes please!!!

Reply:
We do have pants.backend.project_info. I think that might be more natural place for these.